tests/test_ilastik/test_applets/pixelClassification/testPixelClassificationGui.py

```diff
@@ -678,3 +679,76 @@ def impl():
 
         # Run this test from within the shell event loop
         self.exec_in_shell(impl)
+
+    def test_91_suggestFeaturesDlg_close_no_change(self):
+        """
+        Click the "interactive mode" checkbox and see if any errors occur.
+        """
+
+        def impl():
+            workflow = self.shell.projectManager.workflow
+            pixClassApplet = workflow.pcApplet
+            gui = pixClassApplet.getMultiLaneGui()
+
+            gui.currentGui()._labelControlUi.suggestFeaturesButton.click()
+            wait_until(lambda: isinstance(QApplication.instance().activeModalWidget(), SuggestFeaturesDialog))
+            dlg = QApplication.instance().activeModalWidget()
+            dlg.accept()
+
+        # Run this test from within the shell event loop
+        self.exec_in_shell(impl)
+
+    def test_92_suggestFeaturesDlg_run(self):
+        """
+        Click the "interactive mode" checkbox and see if any errors occur.
+        """
+
+        def impl():
+            workflow = self.shell.projectManager.workflow
+            pixClassApplet = workflow.pcApplet
+            gui = pixClassApplet.getMultiLaneGui()
+            opFeatures = workflow.featureSelectionApplet.topLevelOperator
+
+            # this time we want to trigger the message box that informs us,
+            # that some sigmas can only be computed in 2D
+            # so we add a sigma that doesn't fit into size of 50 along z:
+            scales = [0.3, 0.7, 15.0]
+            selections = numpy.array(
+                [
+                    [True, False, False],
```

Comment:
A readability idea: use `0` and `1` instead of `False` and `True`, and then `.astype(bool)` the array. Single-digit numbers are shorter and line up better than boolean constants. Could use this trick in other places too.